telemetry/definitions/commonDefinitions.json

```diff
@@ -1273,6 +1273,18 @@
                 }
             ],
             "passive": true
+        },
+        {
+            "name": "deeplink_open",
+            "description": "User requested that a resource be opened in the browser using the deeplink service",
+            "metadata": [
+                {
+                    "type": "source"
```

Comment:
I assume "Type" represents where within the Toolkit that a link resided.
Any value in adding an optional "Service" field, to determine which services are more frequently used with deeplinks?

Reply:
the deeplinks team can give us that data, so i'd rather not have every toolkit try to figure out how to parse the service segment out of an ARN